{{cookiecutter.repostory_name}}/envs/prod/docker-compose.yml

```diff
@@ -109,11 +109,12 @@ services:
     env_file: ./.env
     environment:
       - DEBUG=off
+      - FLOWER_TASK_RUNTIME_METRIC_BUCKETS=1,2,3,5,10,20,30,45,60,120,180,240,300,600,inf
     command: celery --app={{cookiecutter.django_project_name}} --broker="${CELERY_BROKER_URL}" flower --basic_auth="${CELERY_FLOWER_USER}:${CELERY_FLOWER_PASSWORD}"
     depends_on:
       - celery-worker
     ports:
-      - 5555:5555
+      - 127.0.0.1:5555:5555
```

Comment:
Why do we need to expose a port here in the first place?

Reply:
Celery flower has basic http auth (so it's quite dumb) or oauth (so it's too clever for me). So not really handy to expose it to the internet. But we want to access flower. So we have `cadm <server> tunnel-flower` command which does SSH tunneling from server to `localhost:5555`. However, since flower on server is dockerized, ssh won't see it unless we expose the port to host.